docs/modules/ROOT/pages/how-tos/change-parameter.adoc

```diff
@@ -0,0 +1,37 @@
+= Change a Parameter
+
+Suppose you want to overwrite an inventory parameter for a cluster or tenant.
+This page assumes that you have already set up access to Lieutenant, so that `commodore catalog compile` generally works from any directory.
+
+. Get the Cluster and Tenant ID from Lieutenant and compile the catalog.
++
+[source,bash]
+----
+export $CLUSTER_ID = <target-cluster-id>
+dir=$(mktemp -d)
+pushd "${dir}"
+commodore catalog compile $CLUSTER_ID
+----
+. Open your editor and edit the files you need to change.
++
+TIP: Most probably the file is in `inventory/classes/<tenant-id>`
+. Navigate into the tenant directory and push a new commit
++
+[source,bash]
+----
+pushd inventory/classes/<tenant-id>
+git status
+git add *
+git commit -m "Update cluster"
```

Comment:
Please don't encourage people to use crappy commit messages

Reply:
Removed the -m flag, that should open an editor instead.